lib/neovim-plugin.nix

```
@@ -36,6 +36,9 @@ with lib;
       hasSettings ? true,
       extraOptions ? { },
       # config
+      configLocation ? [ (if isColorscheme then "extraConfigLuaPre" else "extraConfigLua") ],
```

```suggestion
      configLocation ? if isColorscheme then "extraConfigLuaPre" else "extraConfigLua",
```

Because we use `toList` in the implementation, this is fine as a string. Maybe just add a comment saying it can either be a string or a list?